<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>facerecognizer_guiClass</class>
 <widget class="QMainWindow" name="facerecognizer_guiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>537</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>863</width>
    <height>537</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>863</width>
    <height>537</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>facerecognizer_gui</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/facerecognizer_gui/Resources/back.jpg);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ShowButton">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="CloseCamera">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>465</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>465</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/facerecognizer_gui/Resources/cameraback.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;please enter your name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="name">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="photo">
            <property name="minimumSize">
             <size>
              <width>92</width>
              <height>112</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>92</width>
              <height>112</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/facerecognizer_gui/Resources/photoback.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="TakePhoto">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;state:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>7</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="Train">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;state:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>225</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>225</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>863</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="facerecognizer_gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>TakePhoto</sender>
   <signal>clicked()</signal>
   <receiver>facerecognizer_guiClass</receiver>
   <slot>on_TakePhoto_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>546</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>836</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TakePhoto</sender>
   <signal>released()</signal>
   <receiver>facerecognizer_guiClass</receiver>
   <slot>TakePhoto_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>546</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>684</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Train</sender>
   <signal>clicked()</signal>
   <receiver>facerecognizer_guiClass</receiver>
   <slot>Train_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>546</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>864</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CloseCamera</sender>
   <signal>clicked()</signal>
   <receiver>facerecognizer_guiClass</receiver>
   <slot>CloseCarmera_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>51</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ShowButton</sender>
   <signal>clicked()</signal>
   <receiver>facerecognizer_guiClass</receiver>
   <slot>ShowButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>TakePhoto_clicked()</slot>
  <slot>Train_clicked()</slot>
  <slot>CloseCarmera_clicked()</slot>
  <slot>ShowButton_clicked()</slot>
 </slots>
</ui>
